eidos/eidos_globals.cpp

```diff
@@ -1266,7 +1292,16 @@ bool Eidos_CreateDirectory(std::string p_path, std::string* p_error_string)
 // This is /tmp/ (with trailing slash!) on macOS and Linux, but will be elsewhere on Windows.  Should be used instead of /tmp/ everywhere.
 std::string Eidos_TemporaryDirectory(void)
 {
+	#ifdef _WIN32
+	std::string temp_path;
+	char charPath[MAX_PATH];
+	if (GetTempPathA(MAX_PATH, charPath))
+  		temp_path = charPath;
+	std::replace(temp_path.begin(), temp_path.end(), '\\', '/'); // replace windows slashes
```

Comment:
huh, interesting; the C filesystem APIs expect forward slashes rather than backslashes, even on Windows, then?  or how does this work?

Reply:
This is a bit of a workaround. I found that some SLiM tests failed if I didn't do this. Somewhere in the Eidos code the backslashes get interpreted as escape characters, which causes an error. This is the easy solution because the funny thing is that Windows understands forward slashes in paths perfectly fine and converts them to backslashes automatically. So the easiest thing is to just always use forward slashes, because they work on both unix and Windows.

Reply:
Aha, interesting.  OK, great.  A comment that effect, here where you do the replace, would be great.